src/snowflakes/search.py

```diff
@@ -815,7 +73,7 @@ def collection_view_listing_es(context, request):
     if request.datastore != 'elasticsearch':
         return collection_view_listing_db(context, request)
 
-    return search(context, request)
+    return search(context, request, context.type_info.name)
```

Comment:
This is the code we need to fix from updates to origin/dev origin/master

https://github.com/ENCODE-DCC/encoded/commit/cea642d8d0e915180aa327a9fe9947ad69f1f6bb
https://github.com/ENCODE-DCC/snovault/commit/ae660c00164fa3d4786fabb75bcbc8c971ef58c6